### ISSUE TITLE
Ring o' Experience generating infinite EXP out of nowhere. [Game breaking Exploit.]

### ISSUE THREAD

**Issue by iHeartZigg** [MrComputerGhost/Runic-Dungeons#13]
There's not a lot of info i can give you because its such an odd thing.

If i stand near these coordinates with a Advanced Ring o' Experience equipped, it seems to give me infinite XP over time. (Yet to test with normal ring o' experience.)

X : -244
Z: 1350
Y: 72
I am able to move around a few blocks without losing this effect.

Edit; Tested the same coords in a Single-player game, no experience was gained.
Edit2: It is now giving me EXP while i am in my own base, though nothing has changed within it.

http://puu.sh/hOdnH/bd2837cd0a.jpg
http://puu.sh/hOdoo/06893a3228.jpg
Edit; 4 minutes later: http://puu.sh/hOdNz/3af0daa428.jpg

I started with 26 levels and i am now at 303 and steadily climbing. I tried to take the ring off and that seems to stop it.

I'm on a Server running the Feed the Beast Infinity modpack Version 1.5.1.

Edit: Server has restarted and i am no longer gaining levels.

**Comment by MrComputerGhost** [comment]
Are there any mobs being killed near you? What kind of automation systems do you have? What could be causing experience drops>

**Comment by iHeartZigg** [comment]
I wasn't anywhere near any of my structures except for my bee breeding hut, which does not emit EXP orbs or anything of the sort.

Unfortunately it stopped after a server restart so i can't test anything for you. If it happens again, i will investigate further.

At the time, the only mobs being killed were in my cursed earth farm which is located hundreds of blocks away. The only other farm we have is in the End and it was not chunkloaded at the time.

Edit; Found another spot doing the same thing but the server was restarted before i realized. It was nowhere near ANYTHING.

**Comment by MrComputerGhost** [comment]
Next time this happens? Can I have coordinates? Screenshots of the area please?

**Comment by iHeartZigg** [comment]
Here's a short video of it happening.
https://www.youtube.com/watch?v=WzU68Wo2tKg

Currently being processed by youtube, should be up in a minute or two.
Rendering screwed up the colors but it should be fine.

**Comment by iHeartZigg** [comment]
Did some further testing and it appears that the normal version also has the same problem, but i still don't know for sure if the ring is creating it, or another mod is and the ring is just pulling it.

I put down some waypoints and it appears that the area it happens in is a perfect 18x15x18 cuboid shaped area.

So, my guess is that the ring is 18x15x18 and the xp is invisible but only obtainable by the ring. Now, if the xp is from the mod itself or another mod... i have no idea.

Edit; An admin came on the server and killed all XP orbs with the extra utilities command, no change in the rate of XP.

Edit2: Current location is not the same as before. it is now X:-234 Z:1480 Y:85

Edit3: After logging out and back in, it stopped giving me Experience, area is not chunkloaded.

Edit4: Forgot to mention, tried using an Electromagnet from Ender IO Which pulls EXP from 5 blocks away, it did nothing while flying around near the area.

So, the bug seems to stop happening if the area is unloaded.

**Comment by Rynslet** [comment]
Just wanted to say that I'm currently experiencing this bug. Single Player, FTB Infinity Evolved with extra mods, Runic Dungeons 1.1.6a. I'm using the basic ring, and swiftly gaining experience from nowhere. I don't know if it's connected, but I noticed when it started because I heard a level up chime, and was attacked from behind by a creeper jockey.

It's happening only in a limited area in my base, and when I remove the ring or move far enough away, it stops. Similar to what Zigg experienced, it's a 20x20 area centered on nothing in particular, the Fitted Abyssal Stone block next to the crafting table under the Sawmill. I beleive that's where the creeper jockey climbed over the wall, if it does turn out to be connected.

Screenshots of surrounding area:
http://i.imgur.com/oqYC7gy.png
http://i.imgur.com/KVDnKXF.png

EDIT: 24 hours later and I'm experiencing this bug again on the opposite side of my base. (No Creeper Jockey this time, turns out that wasn't the trigger) I don't know how github works, or how to change this to an open issue or whether you'll see this comment, or if I need to make a new thread..

**Comment by UberAffe** [comment]
I believe I am seeing this problem in Infinity Evolved 2.2.2 with no changes to the mods(not expert mode)

I had a blood magic cursed earth combo that I was using to fill my altar and I forgot to set up a collection for the xp. I was walking past it with the Ring o' Experience on and noticed I was gaining levels pretty fast. Looking into and around the spawning area I can't see any xp orbs and I don't get any xp unless I have the ring on. I have currently gotten about 1700 levels from it. The only reason I am not sure of this being a bug is that it has been running constant for about 2 months with no xp collection.

**Comment by Adonis0** [comment]
I have rediscovered this bug in FTB modpack Infinity Evolved 2.3.5 playing on a server running the serverside mods Lag'b'Gon, Tickdynamics and admincommandtoolbox in addition to the default mods.

It was a 19x15x19 cube centered on the coordinates -78x 67y 0z which was an air block, nothing untoward about it. Adding or breaking a block that was in those coordinates did nothing to disrupt the experience gain. Only removing the ring from the Baubles slot or moving outside of the cube prevented the experience gain

Experience gain was a steady rate throughout all levels gained, (30-31 was the same time as 300-301, which should have taken exponentially more xp)

This was not a visual glitch with the GUI, as I could fill a tank of 128 buckets with liquid xp, which equates to over 50 levels, even though I began at 20.
